receiver/googlecloudspannerreceiver/config.go

```diff
@@ -95,10 +100,18 @@ func (instance Instance) Validate() error {
 	}
 
 	for _, database := range instance.Databases {
-		if database == "" {
-			return errors.New("field \"databases\" contains empty database names")
+		if err := database.Validate(); err != nil {
+			return err
 		}
 	}
 
 	return nil
 }
+
+func (database Database) Validate() error {
+	if database.Name == "" {
+		return errors.New("field \"databases\" contains empty database names")
+	}
+
+	return nil
```

Comment:
Should there also be a role defined here?

Reply:
Empty roles are allowed. Roles are required only when FGAC role is being used to access the DB.